modular_skyrat/modules/new_meteors/code/new_meteors.dm

```diff
@@ -0,0 +1,69 @@
+// 1 tick = 2 seconds
+#define METEOR_WAVE_MIN_NOTICE 210 // 7 minutes (420 seconds)
+#define METEOR_WAVE_MAX_NOTICE 270 // 9 minutes (540 seconds)
```

Comment:
Could do this instead. You would then undef `IN_TICK` at the end of the file, obviously.
```suggestion
/// Each tick lasts two seconds.
#define IN_TICKS * 0.5
#define METEOR_WAVE_MIN_NOTICE 7 MINUTES IN_TICKS
#define METEOR_WAVE_MAX_NOTICE 9 MINUTES IN_TICKS
```